src/main/java/com/saralein/server/Controller/FileController.java

```diff
@@ -0,0 +1,45 @@
+package com.saralein.server.Controller;
+
+import com.saralein.server.request.Request;
+import com.saralein.server.response.FileHelper;
+import com.saralein.server.response.ResponseBuilder;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
```

Comment:
Based on experience, I would recommend that you try to use either `java.io` or `java.nio` for file operations. (I prefer `java.nio`)